pkg/types/aws/platform.go

```diff
@@ -101,6 +101,16 @@ type Platform struct {
 	//
 	// +optional
 	LBType configv1.AWSLBType `json:"lbType,omitempty"`
+
+	// PreserveBootstrapIgnition is an optional field that can be used to make the S3 deletion optional
+	// during bootstrap destroy.
+	// +optional
+	PreserveBootstrapIgnition bool `json:"preserveBootstrapIgnition,omitempty"`
+
+	// BootstrapS3Name is an optional field that can be set to change the name of the S3 bucket
+	// created during bootstrap stage. Defaults to "<cluster_id>-bootstrap"
+	// +optional
+	BootstrapS3Name string `json:"bootstrapS3Name,omitempty"`
```

Comment:
Unless we get an answer for https://github.com/openshift/installer/pull/7288#discussion_r1263702327 I don't think we should allow this name to be set.

@mtulio did you have a use case in mind for allowing bootstrap ignition buckets to be named?

Reply:
or `IgnoreBootstrapBucketDeletion`